lightning/src/ln/onion_utils.rs

```diff
@@ -848,49 +843,95 @@ mod tests {
 		assert_eq!(onion_keys[4].rho, hex::decode("034e18b8cc718e8af6339106e706c52d8df89e2b1f7e9142d996acf88df8799b").unwrap()[..]);
 		assert_eq!(onion_keys[4].mu, hex::decode("8e45e5c61c2b24cb6382444db6698727afb063adecd72aada233d4bf273d975a").unwrap()[..]);
 
-		// Test vectors below are flat-out wrong: they claim to set outgoing_cltv_value to non-0 :/
+		// Packet creation test vectors from BOLT 4 (as of writing this at bolt04/onion-test.json
```

Comment:
Could you insert a Github permalink to the cited `onion-test.json` instead of using "as of writing"?

Reply:
Thanks! Fixed with:
[ba522bf](https://github.com/lightningdevkit/rust-lightning/pull/1413/commits/ba522bfde17067d810481b04d8f6014836a101bf)